spec/lib/dfe_data_tables/categories_loader_spec.rb

```diff
@@ -6,12 +6,14 @@
   let(:table_path) { 'spec/fixtures/files/categories_table.xlsx' }
 
   it 'Will perform under a given time' do
-    expect {
-      DfEDataTables::CategoriesLoader.new(table_path)
-    }.to perform_under(10000).ms.sample(10)
+    expect { DfEDataTables::CategoriesLoader.new(table_path) }
+      .to perform_under(5000).ms
   end
 
   it 'Will upload the categories' do
+    Category.destroy_all
+    Concept.destroy_all
+
```

Comment:
It might be nice to find where the extra 35 categories are coming from and cleaning up there in preference to here, but for now this is good enough to merge

Reply:
I'm afraid it's because the test database is the same as the live database.
I have a change in my local `database.yml` that I can push, it would be useful.